documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -9,7 +9,7 @@ You can mount a Raspberry Pi to the back of the Touch Display using its stand-of
 .A Raspberry Pi connected to the Touch Display
 image::images/GPIO_power-500x333.jpg[Image of Raspberry Pi connected to the Touch Display, width="70%"]
 
-Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB so the side with the silver or gold pins faces away from the display. Then connect the other end of the ribbon cable to the `DISPLAY` port on the Raspberry Pi. The pins should face inward, towards the Raspberry Pi.
+Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB. The silver or gold contacts should face away from the display. Then connect the other end of the ribbon cable to the `DISPLAY` port on the Raspberry Pi. The contacts on this end should face inward, towards the Raspberry Pi.
```

Comment:
Not a ribbon cable.

Reply:
[Current docs](https://www.raspberrypi.com/documentation/accessories/display.html) refer to it as such:
"how to connect both the data (ribbon cable)"
"take extra care when attaching the ribbon cable to ensure it is the correct way round"

What else would we call it?

Reply:
Saw comment from earlier. Will update this article, but should we double-check all our references to ribbon cables since I'm pretty sure the camera docs use that terminology as well.

Reply:
Wikipedia says "FFC is a miniaturized form of [ribbon cable](https://en.wikipedia.org/wiki/Ribbon_cable), which is also flat and flexible." which probably explains why the terms are sometimes used interchangeably.